bin/create-test-app.js

```diff
@@ -19,20 +19,32 @@ const appName = `nightly-app-${today}`
 const appPath = path.join(homeDir, "Desktop", appName)
```

Comment:
Would isolating the test artifacts in a temporary directory we delete on completion or error make sense? We can add a flag for the user to override that.